test/controllers/api/tasks_controller_test.rb

```diff
@@ -11,6 +11,24 @@ class Api::TasksControllerTest < ActionController::TestCase
         @request.env['CONTENT_TYPE'] = 'application/json'
       end
 
+      describe 'GET /api/tasks' do
+        it 'lists all tasks with default sorting' do
+          (1..5).map { FactoryBot.create(:dynflow_task, :user_create_task) }
```

Comment:
This can be done like:
```suggestion
          FactoryBot.create_list(:dynflow_task, 5, :user_create_task) }
```
but it's not a blocker, just a nitpick